blocksuite/framework/std/src/gfx/interactivity/extension/base.ts

```
@@ -144,6 +158,18 @@ export class InteractivityActionAPI {
     };
   }
 
+  onElementResize(
+    handler: (
+      ctx: ActionContextMap['elementResize']['context']
+    ) => ActionContextMap['elementResize']['returnType']
+  ) {
+    this._handlers['elementResize'] = handler;
+
+    return () => {
+      return delete this._handlers['elementResize'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)